### PR TITLE
Add file upload component

### PR DESCRIPTION
**Pull request by fredrikloberg**
Added a file upload component according to the Nordea design

Screenshot 2022-02-22 at 17 25 04

### CODE REVIEW

File: src/Nordea/Components/Feedback.elm [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+
+strings =
+    { feedback =
+        { no = "Tilbakemelding"
+        , se = "Återkoppling"
+        , dk = "Tilbagemelding"
+        }
+
+    -- , feedbackDescription =
```
Comment (hungqt) [inline src/Nordea/Components/Feedback.elm]: Same as the comment above

**Review by hungqt — APPROVED** [review]
Nice with the Elm Review too 🤩 🚀

File: src/Nordea/Components/Feedback.elm [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            , maxHeight (vh 100)
+            , overflow auto
+            ]
+            :: attrs
+        )
+        children
+
+
+
+-- receiptView : { header : String, body : String } -> List (Html msg)
```
[the remark this answers is not in the document]
Comment (hungqt) [reply src/Nordea/Components/Feedback.elm]: Nvm already outdated 🤣

[the remark this answers is not in the document]
Comment (fredrikloberg) [reply src/Nordea/Components/Feedback.elm]: yea, wip code stuck by 🤣

**Pull request merged**